rules/xcodeproj.bzl

```diff
@@ -558,17 +563,26 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
         if target_info.targeted_device_family:
             target_settings["TARGETED_DEVICE_FAMILY"] = target_info.targeted_device_family
 
+        pre_build_scripts = []
+        if len(ctx.attr.additional_prebuild_script) > 0:
+            pre_build_scripts = [{
+                "name": "Additional prebuild script",
+                "script": ctx.attr.additional_prebuild_script,
+            }]
```

Comment:
I'd `.append` here too, just in case `pre_build_scripts` gets changed before this `if` in the future